app/experimenter/experiments/tests/test_models.py

```diff
@@ -1085,6 +1085,17 @@ def test_format_firefox_versions_returns_correct_string(self):
         self.assertEqual(experiment_1.format_firefox_versions, "57.0")
         self.assertEqual(experiment_2.format_firefox_versions, "57.0 to 59.0")
 
+    def test_versions_integer_list_returns_correct_list(self):
```

Comment:
If a test fails, it should be super clear from the fact that the whole test failed exactly what is wrong.  If a single test tests multiple things or cases and then fails, then you have to go digging around to figure out WHICH case inside the test was failing, which adds a step to fixing it.  Is this test testing a single case or multiple cases?